pkg/models/app.go

```diff
@@ -12,3 +12,10 @@ type App struct {
 	DeployedVersions      *[]Version `json:"deployedVersions,omitempty"`
 	DeletedAt             string     `json:"deletedAt,omitempty"`
 }
+
+func NewAppByNameAppID(name, appId string) *App {
```

Comment:
Public functions in go should be commented, with the following format:

```go
// NewAppByNameAppID is a ....
```

Reply:
I will add the comment.